docs/secure-connections/sauce-connect-5/installation/docker.md

```diff
@@ -33,8 +33,27 @@ Here are some benefits/use cases for using containerized Sauce Connect Proxy:
     </details>
 2. To run the Sauce Connect Proxy Docker image, modify and run the script below.
 
+   ```shell
+   #!/bin/bash
+
+   # This command sets your environment variables in your current shell. 
+   # Replace the placeholders with your username and access key.
+   SAUCE_USERNAME="<enter username here>" SAUCE_ACCESS_KEY="<enter access key here>" SAUCE_REGION="us-west" SAUCE_TUNNEL_NAME="my-tunnel-name" \
```

Comment:
I suggest we inline it in docker run for simplicity.
Note `my-tunnel-name` probably needs `<>`